src/server.h

```diff
@@ -51,6 +51,7 @@
 #include <lua.h>
 #include <signal.h>
 #include "hdr_histogram.h"
+#include "hdr_alloc.h"
```

Comment:
i suppose we only need this header in server.c?

p.s. thinking of your PR to the upstream hrd_histogram, i'm not sure if they'll like it that you've taken a project of one C file and added a second one for this purpose.
maybe they'll prefer to add another 30 lines to the other 1100 lines C file.